gradle/publish.gradle

```diff
@@ -1,4 +1,3 @@
-import org.gradle.api.publish.maven.internal.artifact.FileBasedMavenArtifact
 
 publishing {
```

Comment:
Is it possible to also move this to `convention.publication.gradle.kts`? Then the whole file include could go away. I guess maybe it does not recognize `kotlinMultiplatform` there?

Reply:
Yeah I tried that, but kotlinMultiplatform is indeed unknown so I kept it like this

Reply:
OK, it can be cleaned up some time as part of a general cleanup.